browser/src/view/components/Button/Button.scss

```diff
@@ -0,0 +1,37 @@
+@use '../../../styles/common/variables' as *;
```

Comment:
What does this do exactly?

Reply:
I thought it was needed to import the global scss variables to be used in other scss files

Reply:
If you set the global SCSS variables as root level CSS variables, I don't believe you'll need to worry about this (unless css variables are unsupported in scss).